tauri-api/src/path.rs

```diff
@@ -0,0 +1,163 @@
+use std::path::PathBuf;
+use serde_repr::{Serialize_repr, Deserialize_repr};
+
+#[derive(Serialize_repr, Deserialize_repr, Clone, Debug)]
+#[repr(u16)]
+pub enum BaseDirectory {
+  Audio = 1,
```

Comment:
Why is Audio the only one with a "Number"?

Reply:
subsequent values increments from the first one